src/Algebra/Properties/Magma/Divisibility.agda

```diff
@@ -39,7 +39,7 @@ x∣yx : ∀ x y → x ∣ y ∙ x
 x∣yx x y = y , refl
 
 xy≈z⇒y∣z : ∀ x y {z} → x ∙ y ≈ z → y ∣ z
-xy≈z⇒y∣z x y xy≈z = ∣-respʳ-≈ xy≈z (x∣yx y x)
+xy≈z⇒y∣z x y xy≈z = {!∣-respʳ-≈ xy≈z (x∣yx y x)!}
```

Comment:
oops: left in a hole (in something that seems irrelevant?)

Reply:
Oops, wrong branch I think